src/main.js

```diff
@@ -1 +1,65 @@
 'use strict';
+
+init();
+
+function init() {
+  const clock = document.querySelector('.clock');
+  const widthClock = window.getComputedStyle(clock, null).width;
+
+  for (let i = 0; i < 11; i++) {
```

Comment:
```suggestion
  for (let i = 1; i < MARKS_COUNT; i++) {
```
Чому 11? Що значить це число? Винеси в константу (const MARKS_COUNT = 12;)